lucien-ios/StartMyCollection/StartMyCollectionViewController.swift

```diff
@@ -9,9 +9,48 @@
 import UIKit
 
 final class StartMyCollectionViewController: UIViewController {
+
+    @IBOutlet private weak var addBookButton: UIButton!
+    @IBOutlet private weak var welcomeImageView: UIImageView!
+    @IBOutlet private weak var toDashboardButton: UIButton!
+    @IBOutlet private weak var getStartedDescription: UILabel!
```

Comment:
This needs to have `Label` appended to the end, otherwise it looks like a String or other data type